input/otlp/logs.go

```diff
@@ -212,16 +213,17 @@ func (c *Consumer) convertLogRecord(
 		)
 	}
 
-	if eventDomain == "device" && eventName != "" {
+	if (eventDomain == "device" && eventName != "") || strings.HasPrefix(eventName, "device.") {
```

Comment:
could you add a comment here that notes the version of the otel spec this is accommodating?

Reply:
Sure, it's done now.